build.gradle

```diff
@@ -107,9 +107,12 @@ allprojects {
     }
 
     tasks.withType(JavaCompile) {
-        options.compilerArgs += ['-Werror']
-        // temporarily relax constraints until we can fix all violations
-        options.errorprone.disable 'AlmostJavadoc', 
+        options.errorprone {
+            disableWarningsInGeneratedCode = true
```

Comment:
Isn't this already set by default? Much of our generated code (protobuf) isn't annotated `@Generated`, so it doesn't help there.
I updated the build scripts in the `-protobufs` projects to opt out of error-prone.